test/problem_data.jl

```diff
@@ -0,0 +1,144 @@
+function run_tests_pbdata(T::Type)
+
+    @testset "Creation" begin
```

Comment:
Here we only need to test `load_problem!`

Reply:
Rewritten to use ``loadproblem!``.